ui/raidboss/popup-text.js

```diff
@@ -121,6 +122,48 @@ class PopupText {
             if (!trigger.regex)
               console.error('Trigger ' + trigger.id + ': has no regex property specified');
 
+            // If trigger has no localized regex but does have a replacement
+            // create regex
+            // FIXME: move this to loading data files to save time
+            if (!trigger[regexLocale] && set.timelineReplace) {
```

Comment:
I understand what you're getting at here, but I would prefer to do this translation not at runtime and commit the values.  That lets other people look at what's going on and commit changes if needed.  Committing these translations also makes it clear that somebody has given it a first pass who is actually using those languages.

This is what the `check_timeline.py` script does, in that it appends translated regexes based on timelineReplace, and you can evaluate if it did the right thing and then commit them.